### ISSUE TITLE
When the node name is'constructor', the program shows that the node name repeats.

### ISSUE THREAD

**Issue by China-Tom** [almende/vis#4224]
When I use vis to draw a network, when one of the nodes is named'constructor', the network displays an error.I checked the IDs of other nodes to confirm that there was no duplication. I want to ask why.Is this constructor the key word?

**Comment by mojoaxel** [comment]
> Is this constructor the key word?

Yes, [`constructor` is a reserved word in javascript](https://www.w3schools.com/js/js_reserved.asp).
Still there should not be an error!

I connot reproduce this issue:
https://jsbin.com/sefefiliqi/edit?html,js,output

**Comment by mojoaxel** [comment]
Looks like a bug to me!

The [documentation](http://visjs.org/docs/network/nodes.html) for `node.id` states:
> The id of the node. The id is mandatory for nodes and they have to be unique. This should obviously be set per node, not globally

Here is nothing that states the `id` can not be a reserved word. We should at least fix the documentation.

**Comment by mojoaxel** [comment]
**Workaround for now**: Don't use [reserved words](https://www.w3schools.com/js/js_reserved.asp)!

**Comment by China-Tom** [comment]
Well, thank you. I see. Thank you very much.
